doc/md/cloud/setup-ci.mdx

```diff
@@ -174,12 +174,12 @@ GitHub CLI extension that will do most of the work for you:
    ```shell
    brew install gh
    ```
-   See [here](https://cli.github.com/manual/installation) for more installation options.
+   See [here](https://github.com/cli/cli#installation) for more installation options.
 
 2. Install the Atlas GitHub CLI extension:
 
    ```shell
-   gh extension install ariga/atlas
+   gh extension install ariga/gh-atlas
```

Comment:
I don't think this is correct

Reply:
it is:
![image](https://github.com/ariga/atlas/assets/62568565/e7ebac2c-926d-4d9a-a848-6b71ada4bb43)